package.json

```diff
@@ -34,15 +34,18 @@
     "pandoc-doc-install": "pandoc --wrap=preserve --columns=10000 --from=markdown --to=rst --output=doc/install.rst ccxt.wiki/Install.md",
     "pandoc-doc-exchanges": "pandoc --wrap=preserve --columns=10000 --from=markdown --to=rst --output=doc/exchanges.rst ccxt.wiki/Exchange-Markets.md",
     "pandoc-doc-exchanges-by-country": "pandoc --wrap=preserve --columns=10000 --from=markdown --to=rst --output=doc/exchanges-by-country.rst ccxt.wiki/Exchange-Markets-By-Country.md",
-    "postinstall": "opencollective postinstall && node postinstall.js"
+    "postinstall": "opencollective postinstall && node postinstall.js",
+    "snyk-protect": "snyk protect",
+    "prepublish": "npm run snyk-protect"
   },
   "types": "./ccxt.d.ts",
   "dependencies": {
     "cloudscraper": "1.4.1",
     "crypto-js": "3.1.9-1",
     "fetch-ponyfill": "4.1.0",
```

Comment:
We have moved these deps to static vendored folders, could you please clean package.json to bring it in line with the current version from the repo? Thx! We might also need to vendor snyk for security reasons...